src/main/scala/WikiTextHelper.scala

```diff
@@ -34,17 +35,42 @@ object WikiTextHelper {
 
   val wikiLinkPattern: Regex = """\[\[(.+?)\]\]""".r
```

Comment:
No changes, maybe worth mentioning that this is used to extract article link in paragraphs, in this format. `[[Text|EntityTitle]]`